contracts/fee_splitter/src/testing.rs

```diff
@@ -1 +1,174 @@
+use bindings_test::{mock_coreum_deps, CoreumApp};
+use coreum_wasm_sdk::core::CoreumMsg;
+use cosmwasm_std::{
+    testing::{mock_env, mock_info, MOCK_CONTRACT_ADDR},
+    Addr, Attribute, BankMsg, Coin, CosmosMsg, Decimal, ReplyOn, Uint128,
+};
+use cw_multi_test::{ContractWrapper, Executor};
 
+use crate::{
+    contract::{execute, instantiate},
+    msg::{ExecuteMsg, InstantiateMsg, QueryMsg},
+    state::Config,
+};
+
+pub type SubMsg = cosmwasm_std::SubMsg<CoreumMsg>;
+
+#[test]
+fn init_works() {
+    let mut app = CoreumApp::default();
+
+    let code_id = store_fee_splitter_code(&mut app);
+    let sender = "addr0000";
+
+    let first_tupple = ("tokenA".to_string(), Decimal::from_ratio(1u128, 2u128));
+    let second_tuple = ("tokenB".to_string(), Decimal::from_ratio(1u128, 2u128));
+    let msg = InstantiateMsg {
+        addresses: vec![first_tupple.clone(), second_tuple.clone()],
+        cw20_contracts: vec!["cw20_contract_one".to_string()],
+    };
+
+    let fee_splitter_instance = app
+        .instantiate_contract(
```

Comment:
- [ ] remove `App`
- [ ] fix arguments to the addresses' list
- [ ] use some actual denoms instead of strings like "cw20_contract_one" to keep the context within the test